WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -264,9 +249,6 @@
 /* In Refund Confirmation, The message shown to the user to inform them that they have to issue the refund manually. The %1$@ is the payment method like “Stripe”. */
 "A refund will not be issued to the customer. You will need to manually issue the refund through %1$@." = "A refund will not be issued to the customer. You will need to manually issue the refund through %1$@.";
 
-/* No comment provided by engineer. */
-"A refund will not be issued to the customer. You will need to manually issue the refund through Stripe." = "A refund will not be issued to the customer. You will need to manually issue the refund through Stripe.";
```

Comment:
Comes from the unit tests, so shouldn't be sent for translation. 🔥 

https://github.com/woocommerce/woocommerce-ios/blob/0073cca68e5a6bc351b51227780d1ce415ae8bac/WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue%20Refund/RefundConfirmationViewModelTests.swift#L157-L158